### ISSUE TITLE
Staged changes does not expand

### ISSUE THREAD

**Issue by arpithindukuri** [microsoft/vscode#201495]
Type: <b>Bug</b>

In the source control pane, the staged changes (incoming/outgoing changes) dropdown does not work.

Steps to reproduce:
Make changes to a github project
Add a commit message, and click commit
When VSCode asks you to stage all changes, click yes
You should see a dropdown saying "incoming/outgoing changes"

Expected behaviour is that clicking on the dropdown should show a list of the staged changes. Clicking on it does nothing for me.

P.S. love VSCode very much, thank you to the team for the great work!

VS Code version: Code 1.85.1 (0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2, 2023-12-13T09:49:37.021Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.89GB (0.40GB free)|
|Process Argv|--crash-reporter-id 067d8df1-24c9-40c0-bebc-f040f97ad805|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (36)</summary>

Extension|Author (truncated)|Version
---|---|---
markdown-preview-github-styles|bie|2.0.3
tailwindshades|bou|0.0.5
vscode-tailwindcss|bra|0.10.4
doxdocgen|csc|1.4.0
postcss|css|1.0.9
dart-code|Dar|3.78.2
flutter|Dar|3.78.0
vscode-typescript-exportallmodules|eli|2.7.0
prettier-vscode|esb|10.1.0
copilot|Git|1.143.0
copilot-chat|Git|0.11.1
material-color|guo|1.4.3
vscode-drawio|hed|1.6.6
jbockle-format-files|jbo|3.4.0
vscode-styled-components|jpo|1.6.6
dart-import|lua|0.3.1
color-picker|Mar|1.0.0
git-graph|mhu|1.30.0
vscode-dotnet-runtime|ms-|2.0.0
remote-ssh|ms-|0.107.1
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.8
remote-explorer|ms-|0.4.1
vsliveshare|ms-|1.0.5900
vscode-paste-image|mus|1.0.4
color-highlight|nau|2.5.0
indent-rainbow|ode|8.3.1
material-icon-theme|PKi|4.32.0
vscode-xml|red|0.26.1
vscode-yaml|red|1.14.0
vscode-taskexplorer|spm|2.13.2
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.55.0
vscode-java-pack|vsc|0.25.15
markdown-all-in-one|yzh|3.5.1

(3 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263cf:30335440
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263:30899288
vscaat:30438848
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyonecf:30548226
f6dab269:30613381
showlangstatbar:30737416
962ge761:30917236
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
dsvsc013:30795093
dsvsc014:30804076
dsvsc015:30845448
pythontestfixt:30902429
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
aa_t_chat:30882232
cp7184t:30925682

```

</details>

**Comment by arpithindukuri** [comment]
Hello @lszomoru, yes!! this is fixed in the current insiders release (v1.86.0-insider (user setup))

![image](https://github.com/microsoft/vscode/assets/60018845/4f83f41b-7c4c-4bc1-adbc-2e6c8c048e47)

Earlier, that "master" dropdown would not open. Thank you!

**Comment by lszomoru** [comment]
@arpithindukuri, thanks for the update. I have closed this issue.
